pkgs/applications/emulators/wine/sources.nix

```diff
@@ -105,10 +117,13 @@ in rec {
   staging = fetchFromGitHub rec {
     # https://github.com/wine-staging/wine-staging/releases
     inherit (unstable) version;
```

Comment:
Shouldn't we have `version = "8.6.1";` here instead of the `rev` hack below? The way you have it, the Nix version of wine-staging will be 8.6, despite using the 8.6.1 release.

Reply:
Yeah... I'm not too happy about it either way. Originally I just wanted to update stable because that is the only wine I use myself at the moment, so I'm also happy to strip the 8.6 update entirely and wait for 8.7.

Reply:
What if we changed the problematic assert to check that the versions are equal *or* `lib.hasPrefix "${lib.getVersion wineUnstable}." patch.version`? That might be closer to the true meaning of staging versions.

Reply:
I've done my fair share of wine updates and I've never had to violate the assert before, so I think it's a very rare occurence. I'm happy to be overruled but my first take was to leave the assert alone.